documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -145,37 +145,38 @@ apiVersion: forklift.konveyor.io/v1beta1
 kind: NetworkMap
 metadata:
   name: <network_map>
-  namespace: {namespace}
+  namespace: <namespace>
 spec:
   map:
     - destination:
-        name: <pod>
-        namespace: {namespace}
-        type: pod <1>
-      source: <2>
-        id: <source_network_id> <3>
+        name: <network_name>
+        namespace: <namespace> <1>
```

Comment:
looking at it again, one line below we write `type: pod` and add that namespace with a comment that it's required only when the type is multus :)
let's drop that line and add the aforementioned comment to <6> (will be <5> after the change)?

Reply:
Done.